packages/broker-service/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Use the official Node.js 20 slim image as a base
+FROM orochinetwork/node:20-iron
+
+# Create a non-root user and group early in the Dockerfile
+RUN groupadd -r appuser && useradd -r -g appuser -s /bin/bash -d /usr/src/app appuser
+
+# Set the working directory
+WORKDIR /usr/src/app
+
+# Ensure the non-root user owns the working directory
+RUN chown -R appuser:appuser /usr/src/app
+
+# Switch to the non-root user
+USER appuser
+
+# Copy the application code as the non-root user
+COPY --chown=appuser:appuser . .
+
+# Install dependencies with a frozen lockfile
+RUN yarn install --frozen-lockfile
+
+# Build the necessary workspaces
+RUN yarn workspace @zkdb/storage build && \
```

Comment:
Later on, pls try to add `yarn install --production --frozen-lockfile` to reduce image size. I think we would make another PR to enhance this.